examples/cloudwatch-logs/README.md

```diff
@@ -14,10 +14,10 @@ module "cloudwatch_logs" {
   source = "coralogix/aws/coralogix//modules/cloudwatch-logs"
 
   coralogix_region   = "Europe"
-  CustomDomain       = "https://<your custom doamin>/api/v1/logs"
+  custom_domain       = "https://<your custom doamin>/api/v1/logs"
```

Comment:
indentation

Reply:
If you have coralogix_region configured no need for custom_domain and vise-versa

Reply:
We've been instructed to support the simple "Region" selector to populate the domain for normal customers, but we need the custom_domain support option for allowing customers to provide their dedicated or privatelink domain endpoints or proxies when needed.